.gitattributes

```diff
@@ -1 +1,4 @@
 * text=auto eol=lf
+*.png -text
+*.waff -text
```

Comment:
I have just noticed a typo: it should be `*.woff`
@ravenx99 Could you make a new PR?